.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: cpp
+
+# Use the faster container-based infrastructure.
+sudo: false
+
+# Use Trusty beta build environment.
+dist: trusty
```

Comment:
Travis' default containerized image ships with an older Ubuntu, and relevantly a very old GCC (4.6.3) and Clang (3.4). These compilers fail on envytools as they don't fully support C++11.

So the options were either to (a) add to envytools' travis.yml lots of configuration commands to update to more modern compiler releases; or (b) base off Travis' more modern Ubuntu images that have the updated compilers already.

The (b) approach chosen here is far more terse and also faster on each run to achieve the same outcome.

Note: Travis don't yet provide containerized images more modern than trusty.